src/XrdHttp/XrdHttpUtils.cc

```diff
@@ -149,32 +150,105 @@ void Tobase64(const unsigned char *input, int length, char *out) {
   return;
 }
 
+void Tobase64(const std::vector<uint8_t> & input, std::string & base64Output) {
```

Comment:
I'd prefer to not add duplicate code if possible. It's probably better to change the existing function and migrate code using it to not use pointers than add this one.